### PR TITLE
Added Darwin Gosal in the mentors #63

### PR DESCRIPTION
**Pull request by lapmid**

### CODE REVIEW

**Comment by viveksb007** [comment]
@lapmid Please squash your commits and try to finalise work before pushing. That's way more commits for this simple issue.

**Comment by lapmid** [comment]
Sorry actually i was mistaken and pushed then i tried to make that correct
so thats why too much pushes

**Comment by lapmid** [comment]
And again sorry actually little bit beginer on github so i made commits everytime i made a correction

File: index.html [hunk 1]
```diff
@@ -400,6 +401,44 @@ <h3 id="">Georges Khazandar</h3>
...
+                    <!-- Mentor Seven -->
+                    <div class="row">
+                        <div class="col-lg-4 col-sm-6 text-center">
+                            <img class="img-circle img-responsive img-center" src="img/Darwin_Gosal.png" alt="">
+                            <h3>
+				<small><a href="https://github.com/gosaldar">Darvin Gosal</a></small>
+				</h3>
+                        </div>
+                        <div class="col-lg-8">
+                            <h3 id="hongphuc">Darvin Gosal</h3>
```
Comment (CloudyPadmal) [inline index.html]: Change the id to "darvin"

File: index.html [hunk 2]
```diff
@@ -400,6 +401,44 @@ <h3 id="">Georges Khazandar</h3>
...
+                            <h4>CryoWerx, Institute of Systems Science, National University of Singapore, Singapore University of Social Sciences</h4>
+                            <p>Darwin Gosal is the Founder of <a href="http://http://www.cryowerx.com">CryoWerx</a>,an F&B retail platform using cloud-connected smart-kiosk. As a member of <a href="https://www.sitf.org.sg/">SiTF</a>,  the CryoWerx platform brings the disruptive cloud technologies and IoT revolution into the F&B sector,in line with the smart-nation initiative. Prior to CryoWerx, he was the Head of IT at Yale-NUS College where he architected a hybrid cloud infrastructure based on SDDC architecture, which has won the Red Hat APAC Innovation Award and was selected as finalist at the Enterprise Innovators Award 2015.
+
```
Comment (CloudyPadmal) [inline index.html]: Remove the extra spaces and align the <p> tags

File: index.html [hunk 3]
```diff
@@ -400,6 +401,44 @@ <h3 id="">Georges Khazandar</h3>
...
+                            <img class="img-circle img-responsive img-center" src="img/Darwin_Gosal.png" alt="">
+                            <h3>
+				<small><a href="https://github.com/gosaldar">Darvin Gosal</a></small>
+				</h3>
+                        </div>
+                        <div class="col-lg-8">
+                            <h3 id="hongphuc">Darvin Gosal</h3>
+                            <h4>CryoWerx, Institute of Systems Science, National University of Singapore, Singapore University of Social Sciences</h4>
+                            <p>Darwin Gosal is the Founder of <a href="http://http://www.cryowerx.com">CryoWerx</a>,an F&B retail platform using cloud-connected smart-kiosk. As a member of <a href="https://www.sitf.org.sg/">SiTF</a>,  the CryoWerx platform brings the disruptive cloud technologies and IoT revolution into the F&B sector,in line with the smart-nation initiative. Prior to CryoWerx, he was the Head of IT at Yale-NUS College where he architected a hybrid cloud infrastructure based on SDDC architecture, which has won the Red Hat APAC Innovation Award and was selected as finalist at the Enterprise Innovators Award 2015.
```
Comment (CloudyPadmal) [inline index.html]: There are some extra spaces in this section and some misses spaces after commas

Comment (CloudyPadmal) [inline index.html]: Fix the link `<a href="http://http://www.cryowerx.com">`

File: index.html [hunk 4]
```diff
@@ -400,6 +401,44 @@ <h3 id="">Georges Khazandar</h3>
...
+			 <!-- Mentor  Eight -->
+                    <div class="row">
+                        <div class="col-lg-4 col-sm-6 text-center">
+                            <img class="img-circle img-responsive img-center" src="img/altman.jpg" alt="">
+                            <h3>
+				<small><a href="http://www.tvbgone.com/">TV-B-Gone</a></small>
+				</h3>
+                        </div>
+                        <div class="col-lg-8">
+                            <h3 id="hongphuc">Mitch Altman</h3>
```
Comment (CloudyPadmal) [inline index.html]: Change the id to altman

File: index.html [hunk 5]
```diff
@@ -400,6 +401,44 @@ <h3 id="">Georges Khazandar</h3>
...
+                            <h4>Hacker and inventor</h4>
+                            <p>Altman was an early developer of Virtual Reality technologies, working at <a href="https://en.wikipedia.org/wiki/VPL_Research">VPL Research</a>  with <a href="https://en.wikipedia.org/wiki/Jaron_Lanier">Jaron Lanier</a> Altman co-founded Silicon Valley start-up <a href="https://en.wikipedia.org/wiki/3ware">3ware</a> in February 1997 with J. Peter Herz and Jim MacDonald (who is on the advisory board of Cornfield Electronics).
+
```
Comment (CloudyPadmal) [inline index.html]: Extra space can be removed and tags can be aligned!

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Hi! I've requested some changes! :)

**Comment by lapmid** [comment]
Okay i wilk do it

**Comment by lapmid** [comment]
@CloudyPadmal I have done changes there was also some prior wrong id's i
have also fixed that please review

**Comment by CloudyPadmal** [comment]
Hi @lapmid ! Looks nice now! Well done! :D There is merge conflicts in the index.html and be really careful when resolving them. Conflict is due to new addition of developers.

**Comment by lapmid** [comment]
Please help how to resolve them

**Comment by lapmid** [comment]
@CloudyPadmal  i have resolved conflicts kindly review

**Comment by lapmid** [comment]
@CloudyPadmal  kindly review now i have removed conflicts second time now

**Comment by mariobehling** [comment]
Darwin is a contributing mentor in FOSSASIA, but not a core mentor of the project. Therefore he should not be added to the project site.

**Comment by lapmid** [comment]
@mariobehling but i had also added mitch altman  and darwin gosal was written in the issue so i had added him

**Pull request closed without merging**